bindings/node.js/src/kzg.cxx

```diff
@@ -142,9 +142,10 @@ inline uint8_t *get_bytes(
     }
     return array.Data();
 }
-inline Blob *get_blob(const Napi::Env &env, const Napi::Value &val) {
-    return reinterpret_cast<Blob *>(get_bytes(env, val, BYTES_PER_BLOB, "blob")
-    );
+inline uint8_t *get_blob(
+    const Napi::Env &env, const KZGSettings *s, const Napi::Value &val
```

Comment:
This is an ultra nit but I think it's preferable to have `KZGSettings` be the last argument here so that `val` is the second argument and matches all the other `get_bytes32()` and `get_bytes48()` function signatures.

Reply:
I'm okay with ultra nits! Thanks for pointing this out. I appreciate consistency. Fixed in [bdcc3b9](https://github.com/ethereum/c-kzg-4844/pull/366/commits/bdcc3b9851578a4c3b785906b73a14c3cc7ba8dd).